src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -4108,6 +4108,54 @@ For some tips on configuring logs for the Node.js agent, see [Configure Node.js
   </Collapser>
 </CollapserGroup>
 
+## Code level metrics [#code-level-metrics]
+
+The code level metrics configuration settings require [Node.JS agent version 9.7.5 or higher](/docs/apm/agents/nodejs-agent/installation-configuration/update-nodejs-agent/).
```

Comment:
Would it be accurate to hyphenate "code-level metrics"?

Reply:
I'm not sure

Reply:
it seems like the language is mixed from searching this repo for both `code level metrics` and `code-level metrics`